pkg/bisect/bisect.go

```diff
@@ -668,6 +676,14 @@ func (env *env) processResults(current *vcs.Commit, results []instance.EnvTestRe
 	return
 }
 
+func (env *env) isTransientError(rep *report.Report) bool {
+	// If we're not chasing a SYZFATAL error, ignore them.
+	// Otherwise it likely indicates some transient problem with the kernel revision.
+	return env.report != nil &&
+		!strings.HasPrefix(env.report.Title, "SYZFATAL:") &&
```

Comment:
Also better to move to pkg/report.Type.
There is also SYZFAIL, do we want to treat it in the same way?
Do we want to treat Go panics in the same way? But if yes, need to somehow special case gVisor.

It would also be useful to move this predicate somewhere else (pkg/report?) b/c I think we want the same predicate during repro process (if we minimize one crash but get SYZFAIL, it's also wrong, right?) and maybe during patch testing, etc.

Also, should it be called "SameError" rather than "TransientError"? I think it's not really about transient, but rather that we are sure that we get a different bug than the original one... however, for patch testing we do want to report other bugs as well, so maybe a function that compares two reports and returns `(same, transient bool)`.